deploy/helm/kubecf/templates/brain-tests.yaml

```diff
@@ -221,4 +221,26 @@ roleRef:
   apiGroup: "rbac.authorization.k8s.io"
   name: "{{ .Release.Namespace }}-{{ .Release.Name }}-test-cluster-role"
 
+# Cluster role binding for service account "tests-brain" and cluster role "cluster-admin"
+---
+apiVersion: "rbac.authorization.k8s.io/v1"
+kind: "ClusterRoleBinding"
+metadata:
+  name: "{{ .Release.Namespace }}-{{ .Release.Name }}-tests-brain-cluster-admin-cluster-binding"
+  labels:
+    app.kubernetes.io/component: "{{ .Release.Namespace }}-{{ .Release.Name }}-tests-brain-cluster-admin-cluster-binding"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
+    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ include "kubecf.chart" . }}
+subjects:
+- kind: "ServiceAccount"
+  name: "{{ .Release.Name }}-tests-brain"
+  namespace: {{ .Release.Namespace | quote }}
+roleRef:
+  kind: "ClusterRole"
+  apiGroup: "rbac.authorization.k8s.io"
+  name: "cluster-admin"
```

Comment:
We _should_ be using a less privileged cluster role (just enough to do what we need), but it's brain tests so I don't care _that_ much.

I guess make a note in [`values.yaml`] that enabling tests could potentially be insecure, and they should only be turned on for isolated testing?

[`values.yaml`]: https://github.com/cloudfoundry-incubator/kubecf/blob/4dc4256f0b57c5624f84f0d2222d52fc8390f403/deploy/helm/kubecf/values.yaml#L294

Reply:
I am ok with documenting this in values.yaml.
Actually got an ok from Troy when I had the idea to use the big permission hammer here.
I originally tried to simply add permissions until I had all which where needed.
Until the system spat back: `You are trying to grant more permissions than you have`.
That is when I switched to cluster admin to whack this particular rodent.

Reply:
Addressed with commit  8c3639e66e3f7b076403e5bfcbac35c26769fdd1. Tested to work. Well, it was just a comment change.